### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by KatalinaBobowik**
Add in README to new repo. @MattWellie Let me know if there's anything off you see with the new repo startup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# CPG Python Template
+# Transcriptional Adaptation
 
-Template repository based on the [CPG team-docs](https://github.com/populationgenomics/team-docs/blob/main/new_repository.md) content.
+Transcriptional adaptation (TA) is a type of genetic compensation whereby the effects of harmful loss of function (LoF) mutations are mitigated by the upregulation of genes with sequence similarity, or gene paralogs (a review of genetic compensation can be found [here](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006780)). Recent [work by El-Brolosy and colleagues](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707827/) has identified this mechanism occurring in zebrafish and mice, however TA has not been validated as a protective mechanism in humans.
```
Comment (lgruen) [inline README.md]:
```suggestion
Transcriptional adaptation (TA) is a type of genetic compensation whereby the effects of harmful loss of function (LoF) mutations are mitigated by the upregulation of genes with sequence similarity, or gene paralogs (a review of genetic compensation can be found [here](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006780)). Recent [work by El-Brolosy and colleagues](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707827/) has identified this mechanism occurring in zebrafish and mice, however TA has not been validated as a protective mechanism in humans.
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# CPG Python Template
+# Transcriptional Adaptation
 
-Template repository based on the [CPG team-docs](https://github.com/populationgenomics/team-docs/blob/main/new_repository.md) content.
+Transcriptional adaptation (TA) is a type of genetic compensation whereby the effects of harmful loss of function (LoF) mutations are mitigated by the upregulation of genes with sequence similarity, or gene paralogs (a review of genetic compensation can be found [here](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006780)). Recent [work by El-Brolosy and colleagues](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707827/) has identified this mechanism occurring in zebrafish and mice, however TA has not been validated as a protective mechanism in humans. 
 
-This contains python and markdown linting configuration, and a github linting action.
+This project will utilise data from the GTEx consortium to assess whether there is evidence of TA occurring in healthy human populations.
```
Comment (lgruen) [inline README.md]:
```suggestion
This project will utilise data from the GTEx consortium to assess whether there is evidence of TA occurring in healthy human populations.
```

**Review by lgruen — APPROVED** [review]
Are you using VS Code as a Markdown editor? I think it'll automatically clean things like trailing whitespace up and it has a nice built-in side-by-side preview.

**Comment by KatalinaBobowik** [comment]
Thanks @lgruen! I'm using VS Code, but seemed to miss the trailing whitespaces? I do have the linters set up now so that should pick up these small things

**Pull request merged**